src/main/scala/org/neo4j/spark/service/MappingService.scala

```diff
@@ -63,8 +65,8 @@ class Neo4jWriteMappingStrategy(private val options: Neo4jOptions)
     }
   }
 
-  private def keysStrategyConsumer(): MappingBiConsumer = new MappingBiConsumer {
-    override def accept(key: String, value: AnyRef): Unit =
+  private def keysStrategyConsumer(schema: StructType): MappingBiConsumer = new MappingBiConsumer {
```

Comment:
is `(schema: StructType)` necessary?
Can we do this?
```suggestion
  private val keysStrategyConsumer = new MappingBiConsumer {
```